### DISCUSSION TITLE
Installing to a subfolder instead of subdomain?

### DISCUSSION THREAD

**Discussion by nycki93** [requarks/wiki#6354] [Help / Questions] [answered]
The docs say that you can't install Wiki.js to a subfolder like example.com/wiki, instead it must be a dedicated domain like wiki.example.com. Why? Can I work around this with a reverse proxy?

**Comment by fnafox** [comment]
As far as I know it's not possible to install it to a different path, it has to be on its own domain/subdomain. I don't believe it will work correctly (if at all) with a reverse proxy setup.

**Comment by NGPixel** [comment, marked as answer]
As stated in the requirements, you cannot install to a subfolder:
https://docs.requarks.io/install/requirements#domain
https://docs.requarks.io/troubleshooting#errors-when-using-a-subdirectory-reverse-proxy
